integration/gobuilder/util.go

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) 2018 The btcsuite developers
+// Use of this source code is governed by an ISC
+// license that can be found in the LICENSE file.
+
+package gobuilder
+
+import (
+	"fmt"
+	"runtime"
+	"strings"
+
+	"github.com/btcsuite/btcd/integration"
+)
+
+// DetermineProjectPackagePath is used to determine target project path
+// for the following execution of a Go builder.
+// Starts from a current working directory, climbs up to a parent folder
+// with the target name and returns its path as a result.
+func DetermineProjectPackagePath(projectName string) string {
```

Comment:
Addresses the https://github.com/btcsuite/btcd/commit/f5b60801a5a706984571bdc25cc228be793f61e0